docs/content/preview/api/ysql/user-defined-subprograms-and-anon-blocks/language-plpgsql-subprograms/plpgsql-syntax-and-semantics/executable-section/compound-statements/loop-exit-continue/_index.md

```diff
@@ -35,10 +35,10 @@ showRightNav: true
 
```

Comment:
The pages in this section (Loop, exit, and continue statements) all have broken links as follows:

URL        `../../basic-statements/cursor-manipulation/'
Name       `Cursor manipulation in PL/pgSQL'
Parent URL  /loop-exit-continue/infinite-and-while-loops/, 
Real URL   https://deploy-preview-18718--infallible-bardeen-164bc9.netlify.app/preview/api/ysql/user-defined-subprograms-and-anon-blocks/language-plpgsql-subprograms/plpgsql-syntax-and-semantics/executable-section/compound-statements/basic-statements/cursor-manipulation/
Check time 0.430 seconds
Result     Error: 404 Not Found

URL        `../../../executable-section/basic-statements/get-diagnostics/'
Name       `The "get diagnostics" statement'
Parent URL  /loop-exit-continue/infinite-and-while-loops/, 
Real URL   https://deploy-preview-18718--infallible-bardeen-164bc9.netlify.app/preview/api/ysql/user-defined-subprograms-and-anon-blocks/language-plpgsql-subprograms/plpgsql-syntax-and-semantics/executable-section/executable-section/basic-statements/get-diagnostics/
Check time 0.605 seconds
Result     Error: 404 Not Found
10 threads active,  2675 links queued, 14470 links in 17176 URLs checked, runtime 13 minutes, 22 seconds

URL        `../../../../../../cursors/#beware-issue-6514'
Name       `Beware Issue #6514'
Parent URL  /loop-exit-continue/infinite-and-while-loops/,
Real URL   https://deploy-preview-18718--infallible-bardeen-164bc9.netlify.app/preview/api/ysql/user-defined-subprograms-and-anon-blocks/cursors/#beware-issue-6514
Check time 0.437 seconds
Result     Error: 404 Not Found

URL        `../../../../../../datatypes/type_array/functions-operators/array-fill/'
Name       `array_fill()'
Parent URL  /loop-exit-continue/array-foreach-loop/,
Real URL   https://deploy-preview-18718--infallible-bardeen-164bc9.netlify.app/preview/api/ysql/user-defined-subprograms-and-anon-blocks/datatypes/type_array/functions-operators/array-fill/
Check time 0.822 seconds
Result     Error: 404 Not Found

URL        `../../../../../../the-sql-language/with-clause/with-clause-syntax-semantics/'
Name       `WITH clause—SQL syntax and semantics'
Parent URL /loop-exit-continue/query-for-loop/,
Real URL   https://deploy-preview-18718--infallible-bardeen-164bc9.netlify.app/preview/api/ysql/user-defined-subprograms-and-anon-blocks/the-sql-language/with-clause/with-clause-syntax-semantics/
Check time 0.971 seconds
Result     Error: 404 Not Found

URL        `../../../../../../datatypes/type_array/functions-operators/properties/#array-lower'
Name       `array_lower'
Parent URL /loop-exit-continue/array-foreach-loop/,
Real URL   https://deploy-preview-18718--infallible-bardeen-164bc9.netlify.app/preview/api/ysql/user-defined-subprograms-and-anon-blocks/datatypes/type_array/functions-operators/properties/#array-lower
Check time 1.341 seconds
Result     Error: 404 Not Found

URL        `../../../../../../datatypes/type_array/literals/'
Name       `Creating an array value using a literal'
Parent URL /loop-exit-continue/array-foreach-loop/,
Real URL   https://deploy-preview-18718--infallible-bardeen-164bc9.netlify.app/preview/api/ysql/user-defined-subprograms-and-anon-blocks/datatypes/type_array/literals/
Check time 1.586 seconds
Result     Error: 404 Not Found

URL        `../../../declaration-section/#plpgsql-bound-refcursor-declaration'
Name       `Semantics for the "plpgsql_bound_refcursor_declaration" rule'
Parent URL /loop-exit-continue/query-for-loop/,
Real URL   https://deploy-preview-18718--infallible-bardeen-164bc9.netlify.app/preview/api/ysql/user-defined-subprograms-and-anon-blocks/language-plpgsql-subprograms/plpgsql-syntax-and-semantics/executable-section/declaration-section/#plpgsql-bound-refcursor-declaration
Check time 1.695 seconds
Result     Error: 404 Not Found

URL        `../../../../../../datatypes/type_array/functions-operators/array-agg-unnest/#unnest'
Name       `unnest()'
Parent URL /loop-exit-continue/array-foreach-loop/,
Real URL   https://deploy-preview-18718--infallible-bardeen-164bc9.netlify.app/preview/api/ysql/user-defined-subprograms-and-anon-blocks/datatypes/type_array/functions-operators/array-agg-unnest/#unnest
Check time 1.644 seconds
Result     Error: 404 Not Found

Reply:
(Thanks for using your link-checker in the _infallible-bardeen_ preview env!) All fixed now. The cause was that I'd moved all of these files one level deeper in the directory tree. And I'd forgotten to do a global search for all outgoing x-refs (very hard to do by hand) to add an extra `../`.